sim.html

```diff
@@ -173,6 +181,12 @@
 				font-size: 8px;
 			}
 			.controlpanel {
+				position: fixed;
```

Comment:
It overlaps the device when I have the dev tools open. You can let it flow below the device or put it in its own box on the side but no fixed or absolute position.